state/core_access.go

```diff
@@ -624,7 +629,8 @@ func (ca *CoreAccessor) RevokeGrantFee(
 	granter := signer.Address()
 
 	msg := feegrant.NewMsgRevokeAllowance(granter, grantee)
-	return signer.SubmitTx(ctx, []sdktypes.Msg{&msg}, user.SetGasLimit(gasLim), withFee(fee))
+	resp, err := signer.SubmitTx(ctx, []sdktypes.Msg{&msg}, user.SetGasLimit(gasLim), withFee(fee))
+	return unsetTx(resp), err
```

Comment:
Could it be the case that problems comes from the fact, that error is not checked before returning, which might result in both `err` and `resp` being non-nil on return. OpenRPC has limitation to return either value or error, so it might lead to problems. Perhaps err != nil check would help?

```suggestion
	resp, err := signer.SubmitTx(ctx, []sdktypes.Msg{&msg}, user.SetGasLimit(gasLim), withFee(fee))
	if err != nil{
		return nil, err
	}	
	return resp, nil
```

Reply:
But this is how it worked previously. unsetTx removes not the entire response but a single field inside.